configuration/product_variant/labs.csv

```diff
@@ -0,0 +1,5 @@
+id,name,categ_id/id,type,purchase_ok,sale_ok,invoice_policy
+init.707688af-bab8-41ae-8d4e-a64f83e9411a,H. Pylori,product.categ_services_lab_test_orders,service,FALSE,TRUE,Ordered quantities
```

Comment:
Isn't there also a product of the same name at the Haiti level? If so, that may become confusing for users.

Reply:
Done, had forgotton to add the Suffix (Selles) to make it distinct, but all good now